src/resources/event/event.model.js

```diff
@@ -0,0 +1,29 @@
+import mongoose from 'mongoose';
+
+const eventSchema = new mongoose.Schema({
+  title: {
+    type: String,
+    required: true,
+  },
+  image: {
+    type: String,
+    required: true
+  },
+  date: { 
+    type: Date,
+    required : true,
+    validate: {
+      validator: function(input) {
+        return new Date(input) >= new Date()? true:false;
+      },
+      message: input => `${input} must be greater than or equal to the current date!`
+    }
+  },
+  body:String,
+  facebook: String,
+  linkdin: String,
+  twitter: String,
+  bookmark : String
```

Comment:
I think these links might just be functionality on the frontend, e.g. to copy the link to a Tweet or whatever. So I think they don't need anything stored in the backend.

Reply:
Correct, you're sharing this resource elsewhere so you don't need these.